src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbTemplate.java

```diff
@@ -465,40 +437,54 @@ private boolean isCollectionSupportSortByString(@NonNull DocumentCollection coll
         return indices.stream().anyMatch(isIndexingSupportSortByString());
     }
 
+    private List<Document> findAllDocuments(@NonNull DocumentQuery query, @NonNull Class<?> domainClass,
```

Comment:
Why the method name is named as `findAllDocuments`? Seems it's not limited with `findAll`?